quickwit/quickwit-control-plane/src/indexing_scheduler/scheduling/mod.rs

```diff
@@ -97,7 +97,7 @@ impl<'a> IdToOrdMap<'a> {
     }
 
     fn source_ord(&self, source_uid: &SourceUid) -> Option<SourceOrd> {
-        self.source_uid_to_source_ord.get(source_uid).copied()
+        self.source_uid_to_source_ord.get(source_uid).cloned()
```

Comment:
why?

Reply:
Copy is better. I just performed an aggressive search and replace in the file, and this is collateral damage. I'll revert.